src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3853,4 +3853,10 @@
   <data name="EditInNotepadDescription" xml:space="preserve">
     <value>Edit the selected file in Notepad</value>
   </data>
+  <data name="PropertyDimensionsColon" xml:space="preserve">
+    <value>Dimensions:</value>
+  </data>
+  <data name="syncStatusColumn/HeaderColon" xml:space="preserve">
```

Comment:
Lets make it `StatusWithColon`

also syncStatusColumn/Header can be just `Status` (this is a good time to remove slash and make it PascalCase).